console/v1/types_console_external_log_links.go

```diff
@@ -0,0 +1,40 @@
+package v1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ConsoleExternalLogLink is an extension for customizing OpenShift web console pod log links.
+type ConsoleExternalLogLink struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+	Spec              ConsoleExternalLogLinkSpec `json:"spec"`
+}
+
+// ConsoleExternalLogLinkSpec is the desired console pod log link configuration.
+type ConsoleExternalLogLinkSpec struct {
+	// text is the display text for the link
+	Text string `json:"text"`
+	// hrefTemplate is an absolute secure URL (must use https) for the log link including
+	// variables to be replaced. Supported variables, which must exactly match, include:
+	// - resourceName - name of the resource which containes the logs
```

Comment:
We should probably include the `${ }` syntax in each of these:

```
// - ${resourceName} - name of the resource which containes the logs
// ...
```
To make it explicit (just like the example).